operator/operator.make

```diff
@@ -1,15 +1,9 @@
-OPERATOR_SDK_VERSION=1.6.1
+OPERATOR_SDK_VERSION=1.14.0
 
 # download operator-sdk binary
 _work/bin/operator-sdk-$(OPERATOR_SDK_VERSION):
 	mkdir -p _work/bin/ 2> /dev/null
```

Comment:
Why the output redirection to `/dev/null`?

If there is an error, we want to see it.